networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -190,17 +223,26 @@ def bfs_predecessors(G, source):
     >>> H.add_edges_from([(0, 1), (0, 2), (1, 3), (1, 4), (2, 5), (2, 6)])
     >>> dict(nx.bfs_predecessors(H, 0))
     {1: 0, 2: 0, 3: 1, 4: 1, 5: 2, 6: 2}
+    >>> G = nx.Graph()
+    >>> nx.add_path(G, [0, 1, 2, 3, 4, 5, 6])
```

Comment:
Use ```G = nx.path_graph(7)``` instead of adding the first path (to save one line in the example). not a big deal.